kie-api/src/main/java/org/kie/api/builder/KieBuilder.java

```diff
@@ -39,6 +41,13 @@
     KieBuilder buildAll();
 
     /**
+     * Builds all the KieBases contained in the KieModule for which this KieBuilder has been created
+     * @param classFilters Used to prevent compilation of Java source files.
+     *          A Java source file vetoed by any of the given filters will not be compiled.
+     */
+    KieBuilder buildAll(Collection<KieClassFilter> classFilters);
```

Comment:
Just my $0.02:

Would it make sense to make the filter more general, e.g. for all resource types and not just Java classes? My thinking is that this is a public API method which we can not easily remove/change in the future (backwards compatibility), so it could be useful to make it more general to cover possibly similar cases with different resource types?